packages/ckeditor5-emoji/src/emojipicker.ts

```diff
@@ -138,6 +138,8 @@ export default class EmojiPicker extends Plugin {
 			this.emojiPickerView = this._createEmojiPickerView();
 		}
 
+		this.emojiPickerView.isEmojiPanelOpen = true;
```

Comment:
I am wondering if we could use the binding engine to update the field automatically.

https://ckeditor.com/docs/ckeditor5/latest/framework/deep-dive/observables.html

If it goes well, we could bind the CSS class without defining an additional class property.

Reply:
> If it goes well, we could bind the CSS class without defining an additional class property.

Binds work as expected, but a view instance cannot access the `ContextualBalloon` plugin. Hence, we need two changes:

1. The view registers an onservable and updates its structure based on its value.
2. Bind the observable with `ContextualBalloon` in the controller (a UI plugin).

See: https://github.com/ckeditor/ckeditor5/pull/17999#issuecomment-2696350260 - I made PoC.